pkg/server/log.go

```diff
@@ -0,0 +1,29 @@
+package server
+
+import (
+	"io"
+	"log"
+	"strings"
+)
+
+type FilteredWriter struct {
+	io.Writer
+}
+
+var msgsToFilter = []string{
```

Comment:
I made a mistake, the correct log message of elb health check is `grpc: Server.Serve failed to complete security handshake from "<ip>": EOF` and not `tls: bad certificate`... sorry, I'll fix.